src/components/DetailedAlarmMetricDisplay/index.tsx

```diff
@@ -0,0 +1,98 @@
+import * as React from 'react';
+import MetricCard from '../MetricCard';
+import Colors from '../../constants/Colors';
+import {
+  ParameterCard,
+  ParameterHeader,
+  ParameterHeaderText,
+  MetricView,
+} from './styles';
+import { Parameter } from 'src/interfaces/Parameter';
+import SetParameter from 'src/interfaces/SetParameter';
+import { VentilationMode } from 'src/models/VentilationMode';
+
+export default function DetailedAlarmMetricDisplay({
+  lowerLimit,
+  title,
+  unit,
+  upperLimit,
+  value,
+  setValue,
+}: Parameter) {
```

Comment:
`setValue` is not part of the `Parameter` interface, it's part of `SetParameter` - is this intended?

Reply:
Also, why not just pass in the main `Parameter` object rather than a list of properties?